config/release.sh

```diff
@@ -1,33 +1,43 @@
 #!/usr/bin/env bash
 set -e
 
+# Check if all required libs are present
+if (! which -s git jq gh lerna-changelog lerna semver); then
```

Comment:
Does this mean lerna, lerna-changelog and semver has to be installed globally?

Reply:
Mhmm, outdated, but true it would cause some problems...